studio-docs/source/composition-publish-notification.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Composition Publish Webhook (Preview)
```

Comment:
1. adjust the title capitalization to match the other titles (lower case after the first term)
2. Let's use the same name as in the payload: Composition status

So "Composition status notifications (preview)".